src/main/java/at/gadermaier/argon2/Argon2.java

```diff
@@ -32,12 +37,15 @@
     private Argon2Type type;
 
     private boolean clearMemory = true;
-    private Charset charset = Charset.forName("UTF-8");
+    private static Charset charset = Charset.forName( "UTF-8" );
```

Comment:
The spaces preceding/following the brackets are really uncommon in Java.

Reply:
I totally agree. Adding an Eclipse auto-format save action to the project would help mimimizing those issues.